tokens/finn.no/buttons.yml

```diff
@@ -13,87 +12,7 @@ color:
         _: s-color-border-primary
         hover: s-color-border-primary-hover
         active: s-color-border-primary-active
```

Comment:
I see you kept the primary border tokens, but I couldn't find any examples of `border-[--w-color-button-primary-border]` and so on. Why didn't we deprecate these tokens?

Reply:
Didn't see that they weren't in use... I'll deprecate them as well.

Reply:
https://github.com/warp-ds/css/pull/144